gr-utils/python/modtool/modtool_add.py

```diff
@@ -110,7 +110,9 @@ def setup(self, options, args):
         if not options.license_file:
             self._info['copyrightholder'] = options.copyright
             if self._info['copyrightholder'] is None:
-                self._info['copyrightholder'] = '<+YOU OR YOUR COMPANY+>'
+                self._info['copyrightholder'] = raw_input("Please specify the copyright holder: ")
+                if(len(self._info['copyrightholder'])==0):
```

Comment:
you could improve the check for an empty (or only space characters) using: 
`if(not self._info['copyrightholder'] or self._info['copyrightholder'].isspace()):`